coreos-ostree-importer/coreos_ostree_importer.py

```diff
@@ -148,6 +173,18 @@ def process(self, message: fedora_messaging.api.Message):
             )
 
 
+def assert_dirs_group_writable(repo: str):
+    founderror = False
+    for root, dirs, files in os.walk(KNOWN_OSTREE_REPOS[repo]):
+        statinfo = os.stat(root)
+        if ((statinfo.st_mode & int('0b000000000010000', 2)) == 0):
```

Comment:
We should probably also check `rx` too here. Could just use [`stat.S_IRWXG`](https://docs.python.org/3.8/library/stat.html#stat.S_IRWXG).

Reply:
ok